contracts/SaleHybridToken.sol

```diff
@@ -64,4 +67,17 @@ contract SaleHybridToken is ERC20, Ownable, Maintenance, PresaleConstants, Reent
         _burn(_presale, _amount);
         isBurnedFor[_presale] = true;
     }
-}
\ No newline at end of file
+
+    function activePresaleAddress() external view returns (address) {
+        AlphaPresale alphaPresaleContract = AlphaPresale(alphaPresale);
```

Comment:
Is it necessary to create a new variable?
Use `if (IPresale(alphaPresale).presaleIsActive())`

Reply:
Done